### PR TITLE
Add server and prep for ui

### PR DESCRIPTION
**Pull request by tdoug**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+    <body>
+        Eeeeeyyooo
```
Comment (SaundersB) [inline public/index.html:3]: image

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/oQ6Rc17gwBU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/BYaqh--Uhx0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (SaundersB) [inline public/index.html:5]: what in the Sam Hill are these slacker references?!? lol

File: app.mjs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import path from 'path';
+import { getCommitersByName } from './git-tool-providers/azure-dev-ops.mjs';
+
+const app = express();
+const port = 3000;
+
+app.get('/api/getCommitersByName', async (req, res) => {
+    let commiters = await getCommitersByName();
```
Comment (SaundersB) [inline app.mjs:9]: could get bogged down pulling all commits or even the last chunk ~15k.

Comment (tdoug) [reply app.mjs:9]: I'm gonna put a limit in the UI and the ability to use a date range.

Comment (tdoug) [reply app.mjs:9]: Ok to merge?

**Review by SaundersB — APPROVED** [review]

**Pull request merged**